frontend/javascripts/admin/organization/organization_edit_view.tsx

```diff
@@ -70,14 +89,18 @@ class OrganizationEditView extends React.PureComponent<Props, State> {
     this.setState({
       isFetchingData: true,
     });
-    const { displayName, newUserMailingList, pricingPlan } = await getOrganization(
-      this.props.organizationName,
-    );
+    const [organization, users] = await Promise.all([
+      getOrganization(this.props.organizationName),
+      getUsers(),
```

Comment:
@fm3 should we provide an extra route for this? fetching all users only to know how many are active seems a bit wasteful to me. on the other hand, there are not really big orgs on wk.org (yet).. what do you think?

Reply:
No strong opinion there. I think it’s fine for the moment.

Reply:
More important is that the superusers are filtered out, those should not count towards the quota. I’m not sure this info is even available in the frontend, so that might mean we have to do this in the backend. @hotzenklotz Do you know if the frontend can (and does) filter this out here?

Reply:
as far as I know, `isSuperUser` is available on the type `APIUser`. so, it can be filtered in the frontend.

Reply:
Yes, the superusers are already filtered in the frontend.

Ideally, the current number of users and the max. per plan are part of the organization request. Kind of like stats. The same is true for the storage usage/limits.

Reply:
I am pretty sure we need a new route, e.g. `/pricing/check`. Average wK users do not have the rights to 1) request all users and 2) fetch the all datasets and the associated storage infos.

Reply:
I added a new route `GET  /api/pricing/status`, returning a json object:

pricingPlan (enum/string)
isExceeded (bool)
isAlmostExceeded (bool, stays true when isExceeded is true)

Note that storage is not yet taken into account, as it is not yet measured.

Reply:
`getPricingPlanStatus` is incorporated now, but `getUsers().length` is still effectively used. This means non-admins might see something like "your plan is exceeded, because 2 of 10 users" are registered? or can't this happen?

Reply:
> Non-admins, should not have access to the organization page. The navbar menu should be hidden from them.

Ok, then I'm just confused by your statement:

> I am pretty sure we need a new route, e.g. /pricing/check. Average wK users do not have the rights to 1) request all users and 2) fetch the all datasets and the associated storage infos.

This sounded like we need a mechanism to get the total user count regardless of permissions (which also exists now but it's only used "partially"). But if only admins can see this page, then it shouldn't matter.